src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter.DecisionDagRewriter.cs

```diff
@@ -13,6 +13,7 @@
 using Microsoft.CodeAnalysis.CSharp.Symbols;
 using Microsoft.CodeAnalysis.PooledObjects;
 using Roslyn.Utilities;
+using static Microsoft.CodeAnalysis.CSharp.SyntheticBoundNodeFactory;
```

Comment:
>static [](http://example.com/codeflow?start=6&length=6)

Are we using `static` members from `SyntheticBoundNodeFactory`?

Reply:
Yes, nested type `SyntheticSwitchSection`